src/test/java/uk/gov/companieshouse/pscfiling/api/controller/impl/PscIndividualFilingControllerImplMergeIT.java

```diff
@@ -12,8 +12,7 @@
 import static org.mockito.Mockito.when;
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.patch;
 import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
-import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
```

Comment:
Best to avoid wildcard imports: check your Intellij editor settings `Editor > Code Style > Java > Imports`